scripts/search.py

```diff
@@ -53,7 +53,7 @@ def run_blastp(query_file_path, subject_file_path, e_value_cutoff):
     """
     Runs BLASTP on the given query and subject FASTA files.
 
-    :param query_file_path: The path to the query FASTA file..
+    :param query_file_path: The path to the query FASTA file.
```

Comment:
While making changes to graphing, I also cleaned up a few miscellaneous grammatical things in the code as identified by PyCharm.